internal/compiler/widgets/material-base/std-widgets.slint

```diff
@@ -14,7 +14,8 @@ import { Slider } from "widget-slider.slint";
 import { ComboBox } from "widget-combobox.slint";
 import { ListView, StandardListView } from "widget-listview.slint";
 import { SpinBox } from "widget-spinbox.slint";
+import { StandardTableView } from "widget-table-view.slint";
```

Comment:
@tronical   can we now use `export { StandardTableView } from "widget-table-view.slint";` ?

Reply:
@ogoffart I think so. `export * from "widget-table-view.slint";` works.

Reply:
Right, only re-exporting of all exports is implemented right now (`export * from "..."`), not re-export by name (`export { Foo } from " ... "`). But I'd like to also support that.